jest.config.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   preset: 'ts-jest',
-  testEnvironment: 'node',
+  testEnvironment: 'jsdom',
```

Comment:
Mocking the window location required updating the test env to be jsdom similar to react components setup.

Reply:
Updated https://github.com/hashicorp/web-platform-packages/pull/63/commits/1e51cec2c65480dd1bb587bc8f50c049f7c6e33a